src/content/getting-started/prepare-your-provider-account/aws/_index.md

```diff
@@ -2,11 +2,216 @@
+        - IPv4 CIDR blocks per VPC: **50**
+        - Routes per route table: **200**
+    - Route 53 Resolver
+        - Endpoints per AWS region: **100** (needed if you are creating a [private cluster]({{< relref "/overview/fleet-management/cluster-management/cluster-concepts/private-cluster" >}}))
+    - Elastic IP
+        - New VPC Elastic IP Address Limit per region: **50** (not needed if you are creating a [private cluster]({{< relref "/overview/fleet-management/cluster-management/cluster-concepts/private-cluster" >}}))
+    - Elastic Load Balancers
+        - Application and Classic Load Balancers per region: **100**
+    - Auto Scaling
+        - Auto Scaling Groups per region: **250**
+        - Launch Configurations per region: **500**
+    - S3
+        - Buckets per Account: **1000**
+    - EC2 Spot Instances
+        - For every primary instance type you tend to use spot instances with, set the limit according to your needs.
+    - EC2 Instances
+        - m4.xlarge per region: **250**
+        - m4.2xlarge per region: **250**
+        - m5.2xlarge per region: **250**
+        - other instance types to be used as workers: increase accordingly
+```
+
+__Note__: Please extend the list of EC2 instances to contain the frequently needed types.
+
+When requesting a service quota increase, you will be asked for a description of your use case. Use this text for the following purposes:
+
+```nohighlight
+We intend to run multiple Kubernetes clusters in this account, potentially used
+by various globally distributed teams. We will be creating and deleting new
+clusters frequently.
+
+Every cluster needs its own VPC for security/isolation reasons and its own
+Elastic IP address for the NAT gateway.
+
+Every cluster has at least 1 Auto Scaling Group, but can contain multiple ASGs if
+multiple instance types are requested as cluster nodes. If we count 50
+clusters with up to 5 EC2 instances each, as worker nodes, we need up to 250
+ASGs. To update the ASGs in a rolling manner, we need to duplicate the ASGs
+for a short time during the update, hence the 500 Launch Configurations.
+
+The number of EC2 instances used as worker nodes is supposed to be scaled
+dynamically based on traffic, hence the high numbers of EC2 instances requested.
+```
+
+## Step 2: Permissions
+
+There are two types of IAM roles that need to be created in the AWS account.
+
+First, the controller role which is used by the CAPA controller in the management cluster to provision all infrastructure to manage workload clusters.
+
+Second, the staff role is used by Giant Swarm engineers to access the AWS account for support purposes.
+
+Here are the mandatory steps to create the required IAM resources but we advocate to follow one of the [automated options](https://github.com/giantswarm/giantswarm-aws-account-prerequisites). There you have terraform or cloudformation templates to create the necessary IAM roles and policies. Otherwise, follow the manual steps below.
```

Comment:
```suggestion
Here are the mandatory steps to create the required IAM resources but we advocate to follow one of the [automated options](https://github.com/giantswarm/giantswarm-aws-account-prerequisites). There you have Terraform or CloudFormation templates to create the necessary IAM roles and policies. Otherwise, follow the manual steps below.
```